File: LMS_CMS_Backend/LMS_CMS_DAL/Models/Domains/LMS/LessonLive.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LMS_CMS_DAL.Models.Domains.LMS
{
    public class LessonLive : AuditableEntity
    {
        [Key]
        public long ID { get; set; }
        public int Period { get; set; }
        public string LiveLink { get; set; }
        public string? RecordLink { get; set; }

        [ForeignKey("WeekDay")]
        public long WeekDayID { get; set; }
        public Days WeekDay { get; set; }

        [ForeignKey("Classroom")]
        public long ClassroomID { get; set; }
        public Classroom Classroom { get; set; }

        [ForeignKey("Subject")]
        public long SubjectID { get; set; }
        public Subject Subject { get; set; }
    }
}

File: LMS_CMS_Frontend/src/app/Models/LMS/lesson-live.ts
export class LessonLive {
    constructor(
        public id: number = 0,
        public period: number = 0,
        public liveLink: string = '',
        public recordLink: string = '',
        public weekDayName: string = '',
        public weekDayID: number = 0,
        public classroomName: string = '',
        public classroomID: number = 0,
        public subjectEnglishName: string = '',
        public subjectID: number = 0,
        public subjectArabicName: string = '',
    ) {}
}
